Agents/CopCalculationAgent/README.md

```diff
@@ -0,0 +1,167 @@
+## Description
+This `COP Calculation` agent is designed to calculate the Coefficient of Performance (COP) of the heat pump based on a given temperature with several assumptions, and instantiated in the [The World Avatar] KG according to the [OntoRegionalAnalysis] ontology. Calculation details can be found in the [home page] of this agent. The required data comprimise the temperature data of the Lower Super Output Area (LSOA) in the UK which can be calculated and instantiated by the [LSOAInputAgent]. This data is updated annually. 
+
+This agent should be implemented with other agents using Derived Information Framework. This agent works with [LSOAInputAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/LSOAInputAgent), [ResultedConsumptionCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/ResultedConsumptionCalculationAgent), [UtilityCostCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/UtilityCostCalculationAgent) and [InequalityIndexCalculationAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/InequalityIndexCalculationAgent),to support the analysis of the impact of heat pumps in the UK (Details refer to   **[Preprint 323](https://como.ceb.cam.ac.uk/preprints/323/)** with **[Preprint 281](https://como.ceb.cam.ac.uk/preprints/281/)**). You may refer to the following graph to find what is the role of this agent in these calculations:
```

Comment:
Replace "... (Details refer to..." with "... (for details refer to...". 

Same comment for other readmes if they use the same text.